BackendSector/Routes/classRoutes.js

```diff
@@ -0,0 +1,10 @@
+const classRoutes = require("express").Router()
+const {addClass, enrollStudent, studentFilter} = require("../databaseMiddleware")
+
+classRoutes.post("/", addClass);
+
+classRoutes.post("/enroll", enrollStudent);
```

Comment:
the readme file has specifications on the way these routes were supposed to be implented

Reply:
we needed a `/class/:className/enroll

Reply:
I couldn't help rebelling against the template lol.